.github/workflows/ci.yml

```diff
@@ -39,16 +41,7 @@ jobs:
         with:
           version: ${{ matrix.version }}
           arch: ${{ matrix.arch }}
-      - uses: actions/cache@v1
-        env:
-          cache-name: cache-artifacts
-        with:
-          path: ~/.julia/artifacts
-          key: ${{ runner.os }}-test-${{ env.cache-name }}-${{ hashFiles('**/Project.toml') }}
-          restore-keys: |
-            ${{ runner.os }}-test-${{ env.cache-name }}-
-            ${{ runner.os }}-test-
-            ${{ runner.os }}-
+      - uses: julia-actions/cache@v1
```

Comment:
Is it safe to cache packages here? I had many issues with packages (such as e.g. PyCall) that mutate the package directory in platform-specific ways, and often had to disable it.

Reply:
Did you have issues outside of PyCall? Normally, the artifacts system should be very robust with caching. It has never caused me any problems.

Reply:
No, not artifacts but packages are the problem. And yes, I think other packages, or their dependencies, were problematic as well but I don't remember which.

Reply:
Should that be an issue at `julia-actions/cache`? I think so. `cache-packages` can be set to false if you prefer that

Reply:
Were your issues all related to Conda? See https://github.com/julia-actions/cache/issues/10 for a discussion

Reply:
The problem is also any other package that saves something to the package directory which is not platform independent. So yeah I think only caching artifacts would be a safer default since the errors where quite hard to debug.

Reply:
It's a problem that also occurs for people manually moving depot paths over in HPC environments and that kind of stuff. Hiding the problem silently at the cache action is not a long-term solution.

Reply:
Hiding? Just setting the default to `cache_packages=false` would be much safer IMO. One could still opt in, but there is no way to rule out these issues as long as packages write to their package directory, e.g., to cache stuff, store paths, build stuff etc.